### PR TITLE
PARQUET-1971: Use fixed JDK version(8.0.252) to fix flaky test in github action

### PR DESCRIPTION
**Pull request by wangyum**
Make sure you have checked _all_ steps below.

### Jira

- [ ] My PR addresses the following [Parquet Jira](https://issues.apache.org/jira/browse/PARQUET/) issues and references them in the PR title. For example, "PARQUET-1234: My Parquet PR"
  - https://issues.apache.org/jira/browse/PARQUET-XXX
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain Javadoc that explain what it does

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java: [ '1.8', '11' ]
+        java: [ '8.0.252', '11' ]
```
Comment (gszadovszky) [inline .github/workflows/test.yml:12]: It might worth a comment why are we such specific for the version.

**Review by gszadovszky — CHANGES_REQUESTED** [review]
Just a note about some comments. Otherwise it looks good to me. Thanks a lot.

**Review by gszadovszky — APPROVED** [review]
Let's wait for a final green test

**Comment by wangyum** [comment]
```
2021-02-04T09:35:28.7952069Z Running org.apache.parquet.hadoop.TestMemoryManager
2021-02-04T09:35:28.8355200Z 2021-02-04 09:35:28 WARN MemoryManager - Total allocation exceeds 95.00% (505,046,624 bytes) of heap memory
2021-02-04T09:35:28.8356554Z Scaling row group sizes to 66.67% for 3 writers
2021-02-04T09:35:28.8640621Z 2021-02-04 09:35:28 WARN MemoryManager - Total allocation exceeds 95.00% (505,046,624 bytes) of heap memory
2021-02-04T09:35:28.8641783Z Scaling row group sizes to 66.67% for 3 writers
```

```
2021-02-04T09:51:16.1543137Z Running org.apache.parquet.hadoop.TestMemoryManager
2021-02-04T09:51:16.1837432Z 2021-02-04 09:51:16 WARN MemoryManager - Total allocation exceeds 95.00% (502,556,256 bytes) of heap memory
2021-02-04T09:51:16.1838907Z Scaling row group sizes to 66.67% for 3 writers
2021-02-04T09:51:16.2418687Z 2021-02-04 09:51:16 WARN MemoryManager - Total allocation exceeds 95.00% (502,556,256 bytes) of heap memory
2021-02-04T09:51:16.2420028Z Scaling row group sizes to 66.67% for 3 writers
```

**Comment by wangyum** [comment]
Sorry @gszadovszky It still fail.

**Comment by gszadovszky** [comment]
So, shall we continue with my PR?

**Comment by wangyum** [comment]
Yes, please.

**Pull request closed without merging**